### ISSUE TITLE
Need NetworkManager, not network-manager

### ISSUE THREAD

**Issue by oj2000us** [arismelachroinos/lscript#274]
I'm a noob with an issue. When starting the network manager after being in monitor mode, the command "service network-manager restart" won't restart the service. The command I have to use is, "service NetworkManager restart". I honestly have no clue if it can be changed by me and my inexperience is the issue. Im willing to learn if it is something I can do. Thank you.

**Comment by thetruemystic** [comment]
Yeah, from what I can tell it was called 'network-manager' in the previous kali version but in newer ones it seems to have changed to 'NetworkManager', so the lazy script just needs to be pointed to the newly updated service. Out of curiosity, what version of Kali are you running? I'm on the latest (2020.4)

**Comment by thetruemystic** [comment]
I might try and fix it locally, and then do a pull request and fix it there too - to my knowledge (still kinda new to all this) you just need to update the main `l` file and replaces instances of `network-manager` with `NetworkManager`, which appears there's seven of so that need to be changed

**Comment by thetruemystic** [comment]
I fixed it, you need to replace your `l.bash` code in /bin/lscript with this code: [here](https://hastebin.com/ukuqikahey.bash)
I was trying to make a pull request, but as there are no other branches I can't. Hope this helps!

**Comment by tera-mithril** [comment]
> I fixed it, you need to replace your `l.bash` code in /bin/lscript with this code: [here](https://hastebin.com/ukuqikahey.bash)
> I was trying to make a pull request, but as there are no other branches I can't. Hope this helps!

Hello, there is no code in the link... could you please check....

**Comment by thetruemystic** [comment]
> > I fixed it, you need to replace your `l.bash` code in /bin/lscript with this code: [here](https://hastebin.com/ukuqikahey.bash)
> > I was trying to make a pull request, but as there are no other branches I can't. Hope this helps!
> 
> Hello, there is no code in the link... could you please check....

I'll try to fix this soon! Don't know what happened there sorry !

**Comment by thetruemystic** [comment]
> > I fixed it, you need to replace your `l.bash` code in /bin/lscript with this code: [here](https://hastebin.com/ukuqikahey.bash)
> > I was trying to make a pull request, but as there are no other branches I can't. Hope this helps!
> 
> Hello, there is no code in the link... could you please check....

Here you go! https://pastebin.com/ny8U1VVD

**Comment by RayxFinkle** [comment]
You can make this easy if you open the "l" file in your preferred text editor, select find and replace, enter Network-Manager and enter NetworkManager in the replace section. It will automatically replace them all for you.
